src/test/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExampleTest.java

```diff
@@ -210,17 +188,91 @@ public void shouldCreateChartsAndAccessThemViaInteractiveQueries() throws Except
     sendPlayEvents(1, songs.get(11), playEventProducer);
 
     playEventProducer.close();
+  }
 
+  private void createStreams(final String host) throws Exception {
+    appServerPort = randomFreeLocalPort();
+    streams =
+        KafkaMusicExample.createChartsStreams(CLUSTER.bootstrapServers(),
+                                              CLUSTER.schemaRegistryUrl(),
+                                              appServerPort,
+                                              TestUtils.tempDirectory().getPath(),
+                                              host);
+    restProxy = KafkaMusicExample.startRestProxy(streams, new HostInfo(host, appServerPort));
+  }
+
+  @After
+  public void shutdown() throws Exception {
+    restProxy.stop();
```

Comment:
should we catch and log an exception here, so we'll still try to close `streams` if `restProxy.stop` fails?

Reply:
Sure, @vvcephei It'll be a better approach. Will apply it.